.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
we should remove this file and add it do `.gitignore`

Reply:
Yes, I will do it.

Reply:
Done